toolkit/offchain/src/plutus_script.rs

```diff
@@ -123,3 +124,9 @@ pub(crate) mod tests {
 		assert_eq!(hex::encode(applied.bytes), hex::encode(CANDIDATES_SCRIPT_WITH_APPLIED_PARAMS));
 	}
 }
+
+impl From<PlutusScript> for cardano_serialization_lib::PlutusScript {
+	fn from(script: PlutusScript) -> Self {
```

Comment:
There is `to_csl` that properly handles language. Please use it here or replace `to_csl` with `From` but one that has complete implementation